recipe/debug_blis.c

```diff
@@ -0,0 +1,36 @@
+#include <blis.h>
```

Comment:
```suggestion
#include <blis/blis.h>
```

Reply:
Noticed the same thing just now while rereading the tests. I had done it on the `-I...` side, but I guess it's cleaner here.